stdlib/array.gr

```diff
@@ -660,6 +728,9 @@ provide let product = (array1: Array<a>, array2: Array<b>) => {
  * @param fn: The function to call on each element, where the returned value indicates if the element satisfies the condition
  * @param array: The array to iterate
  * @returns The total number of elements that satisfy the condition
+ * 
+ * @example
+ * Array.count(a => a >= 7, [> 5, 6, 7, 8, 9]) // 3
```

Comment:
Same As Above:

```suggestion
 * Array.count(a => a >= 7, [> 5, 6, 7, 8, 9]) == 3
```

Reply:
Same comment as before. This is not returning void so simply in line the result.....